### PR TITLE
feat: added rankup command

### PR DESCRIPTION
**Pull request by tooboredtocode**

### CODE REVIEW

**Comment by vcokltfre** [comment]
Use decorators

File: bot/cogs/utility/rankup.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            msg = await ctx.send(f"Select the role you want to give {usr.mention}\n"
+                                 f"Select 1 for Member or 2 for Project Contributor")
+            await msg.add_reaction("1️⃣")
+            await msg.add_reaction("2️⃣")
+
+            def check(reaction, user):
+                return user == ctx.author and str(reaction.emoji) in ["1️⃣", "2️⃣"]
+
+            try:
+                reaction, user = await Bot.wait_for(self.bot, event='reaction_add', timeout=60.0, check=check)
```
Comment (TAG-Epic) [inline bot/cogs/utility/rankup.py]: Why aren't you just using self.bot.wait_for here?

File: main.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "bot.cogs.utility.autopin",
     #"bot.cogs.utility.langwarn",
     "bot.cogs.utility.automod",
+    "bot.cogs.utility.rankup",
```
Comment (TAG-Epic) [inline main.py:13]: what's this indentation?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (tooboredtocode) [reply main.py:13]: wdym?

Comment (TAG-Epic) [reply main.py:13]: Ah just github being broken again sorry

Comment (tooboredtocode) [reply main.py:13]: Lmao, I was gonna scream at py charm for fucking up my PEP 8

**Pull request merged**